Jellyfin.Plugin.Anime/Providers/AniDB/Identity/AniDbTitleDownloader.cs

```diff
@@ -18,7 +18,8 @@ public class AniDbTitleDownloader : IAniDbTitleDownloader
         /// <summary>
         /// The URL for retrieving a list of all anime titles and their AniDB IDs.
         /// </summary>
-        private const string TitlesUrl = "http://anidb.net/api/animetitles.xml.gz";
+        private const string TitlesUrl = "http://anidb.net/api/anime-titles.xml.gz";
+        private const string UserAgent = "jellyfin-plugin-anime";
```

Comment:
I suppose this should be changed everywhere now.

Reply:
I've added the user agent to every request I could find. Unfortunately, `IHttpClient` doesn't seem to have a good way to set headers client-wide, so I've had to include it in each call to `HttpRequestOptions`